packages/components/package.json

```diff
@@ -43,7 +43,6 @@
 		"@wordpress/rich-text": "file:../rich-text",
 		"@wordpress/warning": "file:../warning",
 		"classnames": "^2.2.5",
-		"clipboard": "^2.0.1",
```

Comment:
Why did we removed the package but we still use it on the hook code bellow?

Reply:
The dependency was moved to `compose`: https://github.com/WordPress/gutenberg/pull/22224/files#diff-fbedaa42c7d121aa980384e480772ec4R29, which is already a dependency of `components`.

Reply:
I see, thanks for the clarification @mcsf .